cmd/mock-onelogin/main.go

```diff
@@ -157,6 +163,11 @@ func authorize() http.HandlerFunc {
 			user:     r.FormValue("user"),
 			identity: wantsIdentity,
 		}
+
+		if sub := r.FormValue("sub"); sub != "" {
+			loginWithSub[code] = sub
```

Comment:
could this be added to `sessionData` instead?

Reply:
I don't really like the mock routes doing things outside of the actual API. A better solution might be a separate route to do the setup, or provide a UI to input the option (like the identity bit does)?

Reply:
The UI option makes more sense than entering the UID on the fixtures page as I was struggling to give a route to test this from the start page - I'll add this in 👍